### DISCUSSION TITLE
C64 screen modes

### DISCUSSION THREAD

**Discussion by peter-sarazin** [cc65/cc65#2278] [Q&A]
I would like to be able to create and execute text mode programs for the Commodore 64 using cc65.

I am not sure but it seems like the c64-hi.tgi driver is loaded by default when executing a C64 target executable.

I was hoping to be able to just run in text mode so that I could take advantage of using more than two colors and also use the custom C64 petscii characters.

Is there a way to do this at this time?

Also, I tried calling tgi_uninstall() at the beginning of my program to use text mode, but it just exits the program completely back to a READY prompt.

Here is the repo I created with my sandbox project.
https://github.com/peter-sarazin/c64-cc65-conio-demo

**Comment by polluks** [comment]
Programs in cc65 are using text mode by default (Atari Lynx is an exception).
You don't have to call any TGI function.
